src/ResourceManager/Storage/Commands.Management.Storage/help/New-AzureRmStorageAccount.md

```diff
@@ -222,6 +234,19 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -NetworkRule
+Storage Account NetworkRule```yaml
```

Comment:
@EmmaZhu @blueww please make sure the `` ```yaml `` string is on a line by itself (see other parameters in this file as example). This will allow the markdown file to be rendered correctly.